theano/tensor/tests/test_opt.py

```diff
@@ -5571,6 +5753,20 @@ def test_local_join_empty():
     assert all([not isinstance(n.op, Join) or len(n.inputs) == 4
                 for n in e if isinstance(n.op, Join)])
     assert f.maker.fgraph.outputs[0].dtype == config.floatX
+    
+    # Julian: we can enable the following test, once we 
+    # remove default optimizations.
+    # When we set optimizer=None, no optimizations should be applied,
+    # but that's not the case now...
```

Comment:
I'm not sure I understand this comment. Let's remove that part from the committed code before merging.